### DISCUSSION TITLE
Название файла прошивки

### DISCUSSION THREAD

**Discussion by kilroy98** [shvchk/padavan-builder-workflow#21] [Вопросы и ответы] [answered]
Привет. Заметил что при сборке прошивки через Github Actions в конце названия файла появляются дополнительные символы (в моём случае на последней сборке это 066 - "RM-AC2100_3.4.3.9L-102-9e787ec**066**.trx"), но при сборке через Prometheus их нету. Понимаю что они ни на что не влияют, но интересно значат ли они что-то и откуда берутся.

**Comment by shvchk** [comment, marked as answer]
По-моему, это задано в файле `trunk/versions.inc`:

```sh
FIRMWARE_BUILDS_REV=$(shell git rev-parse --short HEAD 2>/dev/null)
```

и не менялось уже лет 5. Предположу, что разница в выводе команды `git rev-parse` связана со средой исполнения или версией git.
